src/integrationTest/java/uk/gov/hmcts/reform/wataskmanagementapi/services/calendar/DueDateConfiguratorTest.java

```diff
@@ -0,0 +1,362 @@
+package uk.gov.hmcts.reform.wataskmanagementapi.services.calendar;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
+import uk.gov.hmcts.reform.wataskmanagementapi.domain.entities.camunda.CamundaValue;
+import uk.gov.hmcts.reform.wataskmanagementapi.domain.entities.camunda.ConfigurationDmnEvaluationResponse;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+
+@SpringBootTest
+@ActiveProfiles({"integration"})
+public class DueDateConfiguratorTest {
+
+    public static final LocalDateTime GIVEN_DATE = LocalDateTime.of(2022, 10, 13, 18, 00, 00);
+    @Autowired
+    private DueDateConfigurator dueDateConfigurator;
+
+    @Test
+    public void shouldCalculateDueDateWhenMultipleDueDateAndTimesAreAvailable() {
```

Comment:
should we also cover  scenario when it's only multiple dueDateOrigin?

Reply:
public void shouldCalculateDueDateWhenMultipleDueDateOriginsAreAvailable() {